code/api.tf

```diff
@@ -391,6 +409,20 @@ resource "azurerm_api_management_certificate" "jwt_certificate" {
   data                = pkcs12_from_pem.jwt_pkcs12.result
 }
 
+# API Product
+
+resource "azurerm_api_management_product" "cgn_onbording_portal" {
+  product_id            = "cgn-onboarding-portal-api"
+  resource_group_name   = azurerm_resource_group.rg_api.name
+  api_management_name   = module.apim.name
+  display_name          = "CGN ONBOARDING PORTAL API"
+  description           = "CGN Onboarding Portal API"
+  subscription_required = true
+  approval_required     = false
+  published             = true
+}
```

Comment:
AFAIK this conf can be used only in UAT, since target API ( for which we generate Subscription keys ) is exposed through IO ApiM

Reply:
Added some variables to point to the IO ApiM